eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -4365,13 +4365,13 @@ groups:
               - [ "Re", "Im" ]
               - [ "m7", "m5", "m3", "m1", "t0" ]
               - [ "perp", "para", "long"]
-      'B->K^*ccbar::%1%_Hhat_at_%2%_%3%@GRvDV2022' :
+      'B->K^*ccbar::A%1%_Hhat_at_%2%_%3%@GRvDV2022' :
           central: 0
           min:     0
           max:     0
-          latex:   '$\mathrm{%1%}\, \hat{\mathcal{H}}_\%3%^{B\to K^*c\bar{c}}(%2%)$'
+          latex:   '$\mathrm{A%1%}\, \hat{\mathcal{H}}_\%3%^{B\to K^*c\bar{c}}(%2%)$'
           matrix:
-              - [ "Abs", "Arg" ]
+              - [ "bs", "rg" ]
```

Comment:
This works here. Have to think about how to solve this problem in general. Add a special case when the value assigned to ``matrix`` is a map rather than a list? Just brainstorming... Nothing to do here right now.